modules/managed-cluster-configure-notification-types-received.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="config-notification-types-received_{context}"]
+= Configuring notification types received
+
+Notification contacts receive emails when cluster notifications are sent to the cluster.
+
+By default, a notification contact receives an email for every notification sent to the cluster, regardless of notification type. You can configure the type of notification you want to receive in Hybrid Cloud Console notification settings for OpenShift.
+
+.Prerequisites
+* Your cluster is deployed and registered to Hybrid Cloud Console.
```

Comment:
Should it be "the Hybrid Cloud Console"? I'm not sure, but this caught my eye. Or more annoyingly, "Red Hat Hybrid Cloud Console"?

Reply:
All ROSA/OSD clusters are "registered" with OCM. This prereq should be removed.

Reply:
It should, and I should have used the variables instead of writing these out - corrected in all locations now :)